SMLHelper/Logger.cs

```diff
@@ -1,36 +1,80 @@
 ﻿namespace SMLHelper.V2
 {
     using System;
+    using System.IO;
+
+    internal enum LogLevel
+    {
+        Debug,
+        Warn,
+        Error,
+        Info
```

Comment:
If the order of the elements in the enum matters, usually the values should be expressed explicitly
```cs
Debug = 0,
Info = 1,
Warn = 2,
Error = 3,
```

Reply:
They can be, but its not required.